### DISCUSSION TITLE
Transcoding and persistency problems on Unraid

### DISCUSSION THREAD

**Discussion by srvprivate** [mediacms-io/mediacms#806] [Q&A]
**Describe the issue**
A: 1. Media not persistent upon restart (named volumes) (SMB share)
B. Transcode either hang or never complete, esp. with anything over 4 min.
**To Reproduce**
Steps to reproduce the issue:
A: 1. restart media missing yet stll retained in Originals folder.
B. 1. Media is uploaded, transcodes usually show pending and never complete at all. No way to see status except "pending" PLEASE add some sort of percentage or access to ffmpeg's status. I know t's possible. I've used other apps that allow it
**Expected behavior**

A. MEdia should be persistent.
B. Transcodes should complete and status should be shown.
**Screenshots**
A. would just show an empty media library
B. would just show "transcode pending" on the video
**Environment (please complete the following information):**
 - OS: Unraid (slackware)
 - Installation method: Docker Compose
 - All browsers
 -

**Additional context**

**Comment by mgogoulos** [comment]
I've changed the title and moved to discussions, since this looks to affect a specific environment (Unraid)

**Reply by srvprivate** [reply]
Ah, I see. It's running in Docker compose, so can't imagine it's too married to unraid unless there's something weird. Thanks

**Reply by mompl** [reply]
Hi @srvprivate 
I was also looking to get mediacms working on unraid. Unfortunately not under the comunity apps and to be honest, I've not much of a clue to get it done with docker compose.
Do you mind to share some details how you got it setup on unraid?
Would really appreciate that.
Thanks a lot.

**Comment by srvprivate** [comment]
Hello there, I would reallllllllly encourage you to learn Docker Compose, I’m more than happy to help, but lasting experience in that environment will do you so much good! Unraid’s Community Applications are wonderful, however the way I see them from my experience, are somewhat of training wheels and convenience. If one is serious about learning Docker, invest time in learning the syntax of Docker Compose. It is relatively easy to learn! From there, you can grab YAMLs from Github, like the one here for Production, and begin editing the template to fill in the information you need, ie, database passwords, etc. Of course you’ll want the .env file too. That’s really all you need most of the time to get started. 

It was easy to setup using Docker Compose, and you can use the Docker Compose plugin in Unriad, and when creating a new compose file, point your directory to /mnt/user/appdata/FOLDER_NAME — this makes it easier to keep track of your compose files, rather than storing them to USB. (Or wherever you store your application data), some people are on /opt.


￼

Confidentiality and Security Notice: This email was transmitted using advanced OpenPGP encryption technology to ensure secure and private communication. Messages are protected at all stages, helping to safeguard confidentiality. This information is intended solely for the recipient(s) of this email, and any disclosure, copying, or distribution outside of this email thread is strictly prohibited. If you are not the intended recipient, please delete this email and notify the sender immediately. Unauthorized access or misuse is prohibited.
